doc/porting_guide.md

```diff
@@ -297,49 +313,57 @@ The examples are command line applications libxively links against. They require
 
     - [ wolfssl | mbedtls ] - selects the TLS library which is used to provide secure connection with Xively servers, WolfSSL is set by default
 
-### The Porting Step
+### Porting the Xively C Client to Your Platform
+
+#### Let's Begin
+
+As a rule of thumb if you are stuck or confused, then please user existing platform config files like: `make/mt-os/mt-cc3200` or `make/mt-os/mt-linux` files as an example.
 
-#### Cross-compilation with the _make_ build system
+Let's assume the new platform's name is np2000. 
 
-To cross-compile the Xively C Client on OSX or Linux/Unix to a new fictitious platform *np2000* the following command should be enough
+
+#### The Goal: Cross Compilation with the _make_ Build System
+
+The ideal goal to cross-compile the Xively C Client on OSX or Linux/Unix to your platform.  For the sake of this tutorial, let's pretend that your platform is a new fictitious platform: The *NP2000*.  You'r platform sounds awesome.
+
+Our goal is construct the following command to build the Xively Client for the NP2000:
```

Comment:
Our goal is to build the Xively C Client for the NP2000 with the following command:
?